demo.wiretapp/demo.wiretappUITests/WiretappBaseTestCase.swift

```diff
@@ -0,0 +1,222 @@
+import Foundation
+import XCTest
+
+class WiretappBaseTestCase: XCTestCase {
+    let timeOut: TimeInterval = 30
+    var app = XCUIApplication()
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        if let mockResponses = getRootPath() {
+            app.launchEnvironment = [
+                Wiretapp.testCasePath: mockResponses + "/MockResponses/" + name.createDirectoryPath()
+            ]
+            print(mockResponses + name.createDirectoryPath())
+        }
+        app.launch()
+    }
+    
+    func getRootPath() -> String? {
+        let pathArray: [String] = #file.split(separator: "/")
```

Comment:
Neat!

Reply:
I'm wondering.. does this mean the placement of this file has to always be 2 levels deep from the project root?